applications/teletype/src/templates/teletype_port_request_admin.erl

```diff
@@ -0,0 +1,219 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz Inc
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(teletype_port_request_admin).
+
+-export([init/0
+         ,handle_req/2
+        ]).
+
+-include("../teletype.hrl").
+
+-define(TEMPLATE_ID, <<"port_request_admin">>).
+-define(MOD_CONFIG_CAT, <<(?NOTIFY_CONFIG_CAT)/binary, ".", (?TEMPLATE_ID)/binary>>).
+
+-define(TEMPLATE_MACROS
+        ,wh_json:from_list(
+           ?PORT_REQUEST_MACROS
+           ++ ?ACCOUNT_MACROS
+          )
+       ).
+
+-define(TEMPLATE_TEXT, <<"Port request submitted for {{account.name}} by {{port_request.customer_contact}}.\n\nThe account's tree:\n\n {{ account.name }} ({{ account.id }})\n{% for id, name in account_tree %} {{ name }} ({{ id }})\n{% endfor %}\n\nRequest to port numbers: {{ port_request.numbers }}.\n\nPort Details:\n\n {% for k,v in port_request %} {{ k }} : {{ v }}\n {% endfor %}\n">>).
```

Comment:
Yes, .numbers is converted to a binary with `wh_util:join_binary/2` towards the bottom of the file